src/cow-react/modules/limitOrders/containers/TradeButtons/limitOrdersTradeButtonsMap.tsx

```diff
@@ -46,7 +46,7 @@ export function SwapButton({
   onClick?: () => void
 }) {
   return (
-    <ButtonPrimary onClick={onClick} disabled={disabled} buttonSize={ButtonSize.BIG}>
+    <ButtonPrimary fontSize={'16px !important'} onClick={onClick} disabled={disabled} buttonSize={ButtonSize.BIG}>
```

Comment:
Note: `fontSize` here is so that this message is in one line and not like this, but maybe @fairlighteth can improve this in future
![Снимак екрана из „2022-12-21 11-46-23“](https://user-images.githubusercontent.com/34926005/208887399-33ce2be5-bfad-484b-9f48-74e2b75ecc65.png)